aspnetcore/test/troubleshoot-azure-iis.md

```diff
@@ -113,6 +113,8 @@ The [ASP.NET Core Module](xref:host-and-deploy/aspnet-core-module) attempts to s
 
 A common failure condition is the app is misconfigured due to targeting a version of the ASP.NET Core shared framework that isn't present. Check which versions of the ASP.NET Core shared framework are installed on the target machine.
 
+If using Azure Key Vault, another common failure condition is lack of permissions to the Key Vault. Check the access policies in the targeted Key Vault to ensure the website user has access to read secrets. 
```

Comment:
A common failure condition with Azure Key Vault is lack of permissions to the Key Vault. Check the access policies in the targeted Key Vault to ensure the website user has access to read secrets.